docs/integrations/laravel.rst

```diff
@@ -51,7 +51,7 @@ Add your DSN to ``.env``:
 
 .. code-block:: bash
 
-    SENTRY_DSN=___DSN___
+    SENTRY_LARAVEL_DSN=___DSN___
```

Comment:
Laravel forces all `.env` vars in the `$_ENV` which causes problems when wanting to override the DSN from the Laravel config since the SDK itself will pickup `SENTRY_DSN` from `$_ENV`.

See: https://github.com/getsentry/sentry-laravel/issues/105#issuecomment-384162662 and down